jujugui/static/gui/src/app/app.js

```diff
@@ -750,6 +757,27 @@ YUI.add('juju-gui', function(Y) {
     },
 
     /**
+      Renders the Deployment component to the page in the
+      designated element.
+
+      @method _renderDeployment
+    */
+    _renderDeployment: function() {
+      var env = this.env;
+      var ecs = env.get('ecs');
+      var changesUtils = this.changesUtils;
+      var currentChangeSet = ecs.getCurrentChangeSet();
+      React.render(
+        <window.juju.components.Deployment
+          ecsCommit={ecs.commit.bind(ecs, env)}
+          generateAllChangeDescriptions={
```

Comment:
Because this gets re-rendered every time the changeset changes what do you think about passing just the return value of this method instead of passing the method itself?